SmartDeviceLink-iOS/SmartDeviceLink/SDLProtocol.m

```diff
@@ -41,108 +43,124 @@ - (void)logRPCSend:(SDLProtocolMessage *)message;
 @implementation SDLProtocol
 
 - (id)init {
-	if (self = [super init]) {
+    if (self = [super init]) {
         _version = 1;
         _messageID = 0;
         _sessionID = 0;
         _recieveQueue = dispatch_queue_create("com.sdl.recieve", DISPATCH_QUEUE_SERIAL);
         _sendQueue = dispatch_queue_create("com.sdl.send.defaultpriority", DISPATCH_QUEUE_SERIAL);
         prioritizedCollection = [SDLPrioritizedObjectCollection new];
-
+        
```

Comment:
If I had to guess, I would say they were automatic Xcode changes in response to spacing inconsistencies? I can't really tell what happened here.